default.nix

```diff
@@ -0,0 +1,24 @@
+  {}:
+with (import (builtins.fetchTarball {
+  url = "https://github.com/dmjio/miso/archive/1114ba461605cef11ef4e1a96b5adfc4b4c9af18.tar.gz";
+  sha256 = "03x323yjpx3wq87kb2i202cw6sxj3sb79j4h712jiv4yd993gjlz";
+}) {});
+let
+  inherit (pkgs) runCommand closurecompiler;
+  inherit (pkgs.haskell.packages) ghcjs86;
+  client = clientDrv.callCabal2nix "tomland-online" ./. {};
+
+  # project derivation.
+  clientDrv = ghcjs86.extend (pkgs.haskell.lib.packageSourceOverrides {
+    tomland = builtins.fetchTarball "https://github.com/kowainik/tomland/archive/09bf050a3e964ef137e1f96de76f17ac2b6198b4.tar.gz";
+    parser-combinators = builtins.fetchTarball "https://github.com/mrkkrp/parser-combinators/archive/7996964b0f4da5adfed5768656e2f1e47cebe659.tar.gz";
+    });
```

Comment:
Looks like we're going to get a new profession :hammer_and_pick: :construction_worker_woman: